src/AppInstallerCLITests/WorkFlow.cpp

```diff
@@ -1139,7 +1139,6 @@ TEST_CASE("InstallFlow_Zip_Exe", "[InstallFlow][workflow]")
 {
     TestCommon::TempFile installResultPath("TestExeInstalled.txt");
     TestCommon::TestUserSettings testSettings;

```

Comment:
This testUserSettings will not be used after removing, so please clean up across this file